README.md

```diff
@@ -9,3 +9,7 @@ This is for stuff relevant to GrimoireLab as a whole. For example:
 Coordinated releases are snapshots (specific commits)
 of most of the GrimoireLab components that are expected to work together.
 See more information in the [releases README.md file](releases/README.md).
+
+* Utlis (directory [utlis](utils)) for doing stuff relevant to GrimoireLab
```

Comment:
Utlis must be Utils and utlis must be utils

Reply:
Not sure what you are asking for here. The first `Utils` is uppercase because it is just a noun starting a sentence. The second and third `utils` are just the name and links of the `utlis` directory.

Reply:
OK, now I realize. Sorry, I had not noticed the spelling error. I'm fixing it in the new commit.